ydb/core/protos/data_events.proto

```diff
@@ -43,6 +43,17 @@ message TTabletInfo {
     optional bool IsFollower = 5;
 }
 
+message TMvccSnapshot {
+    optional uint64 Step = 10;
+    optional uint64 TxId = 11;
+    // When true (the default) reads from the specified snapshot are expected
+    // to be repeatable, i.e. future reads with the same snapshot version must
+    // observe the same data. For non-repeatable reads (or writes) the above
+    // version is the minimum MVCC version for the operation, shards may use
+    // a higher MVCC version when appropriate.
+    optional bool RepeatableRead = 12 [default = true];
```

Comment:
Don't you want a modal test for `RepeatableRead = false`?

Reply:
It's not implemented yet, so there is nothing to test.